server_tests/test/clientTests.js

```diff
@@ -508,36 +503,45 @@ describe("RUST API TEST SUITE", function() {
             this.timeout(timeoutValue);
             return testClient.takePicture(sessionId)
             .then( function onSuccess (res) {
-                Comparison.oscTakePictureOutput(res);
+                validate.done(res.body, schema.names.commandTakePicture);
                 return testClient.takePicture(sessionId);
             })
             .then( function onSuccess (res) {
-                Comparison.oscTakePictureOutput(res);
+                validate.done(res.body, schema.names.commandTakePicture);
                 return testClient.listImages(1, false);
             })
             .then( function onSuccess (res) {
-                Comparison.oscListImagesOutput(res, true, false, {entries: [{'one': 'one'}], totalEntries: 2});
+                validate.done(res.body, schema.names.commandListImages);
+                assert.notEqual(res.body.results.continuationToken, undefined);
```

Comment:
To be consistent we should probably use `property()` instead of comparing with `undefined`. This also applies to elsewhere in the tests, so check them all.